MaxText/configs/base.yml

```diff
@@ -80,6 +80,7 @@ attention: 'flash' # Supported attention: dot_product, flash, gpu_flash_xla, gpu
 # Combine matmuls for QKV and MLP
 fused_qkv: False
 fused_mlp: False
+combined_kv_cache: True
```

Comment:
BTW MaxText usually doesn't bother with this kind of thing. We just implement what we think is best!

(Not sure either way on this CR but just stylistically we are focused on "only the right path" wherever possible